docs/modules/user-guide/partials/proc_migrating-commands.adoc

```diff
@@ -2,33 +2,36 @@
 = Migrating commands
 
 [role="_abstract"]
-This section describes how to migrate plug-ins to devfile v2. There ARE major changes in the `commands` section of the devfile. A command specified in a v1.0 devfile will NOT work as it is in a devfile v2.0. 
+This section describes how to migrate commands to devfiles v2.x. The command section of devfile v2.x has major changes. A command specified in a devfile v1.x no longer works in a devfile v2.x.
 
 .Procedure
 
-. Command Groups: build,run,test,debug. 
-. Apply Command. 
-. Environment Varibables for a Specific Command. 
-. Renaming workdir into workingDir. 
-. Id and label for Composite Commands. 
-. Run exec Commands as Specific User. 
+. Command groups: build, run, test, debug.
```

Comment:
is it clear to the reader how to command groups? I don't understand this step^^

Reply:
Yes, this doc needs major revisions which we've discussed in previous PRs. We'll tackle these concerns in the separate issue for properly structure our procedures: https://github.com/devfile/api/issues/402